examples/OpenGL45/arrayReductionByAddition_version1.shadertrap

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 Shiyu Liu
```

Comment:
Please change this to:

```
# Copyright 2021 The ShaderTrap Project Authors
```

The same for the other new .shadertrap files.

That will keep the bots happy.

Reply:
Done.